pages/glossary/inert.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Inert
+key: inert
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An [HTML or SVG element][] is inert if:
+- it has an `inert` [attribute value][] of true; or
+- one of its ancestor elements has an `inert` [attribute value][] of true; or
+- it is [blocked by a modal](https://html.spec.whatwg.org/multipage/interaction.html#blocked-by-a-modal-dialog).
```

Comment:
All the references need to be defined in a references list at the bottom of the file.